Gemfile.lock

```diff
@@ -1,9 +1,13 @@
-PATH
-  remote: /Users/dlhaines/dev/GITHUB/dlh-umich.edu/FORKS/google_groups_gem
+GIT
+  remote: https://github.com/dlhaines/google_groups_gem.git
+  revision: c64f3f986031b3eec485ffaa465f09fe05823f8d
```

Comment:
Yes.  This done automatically when running bundle install to update the dependencies.